app/components/teachers/details/past_induction_periods_component.html.erb

```diff
@@ -4,7 +4,13 @@
   <% past_periods.each do |period| %>
     <li>
       <%=
-        govuk_summary_list(card: { title: period.appropriate_body.name, actions: enable_edit == true ? [edit_link(period), delete_link(period)].compact : [] }) do |sl|
+        govuk_summary_list(
+          card: {
+            heading_level: 3,
+            title: period.appropriate_body.name,
+            actions: enable_edit ? [edit_link(period), delete_link(period)] : []
```

Comment:
This can be simplified further as you already `return unless enable_edit` in both `edit_link` and `delete_link`.

```suggestion
            actions: [edit_link(period), delete_link(period)].compact
```